docs/configuration.md

```diff
@@ -284,6 +284,7 @@ the backend workers convert "Bytearray to utf-8 string" when the Content-Type of
 * `model_server_home` : Torchserve home directory. 
 * `max_request_size` : The maximum allowable request size that the Torchserve accepts, in bytes. Default: 6553500
 * `max_response_size` : The maximum allowable response size that the Torchserve sends, in bytes. Default: 6553500
+* `limit_max_image_pixels` : Default value is true (Use default [PIL.Image.MAX_IMAGE_PIXELS](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.MAX_IMAGE_PIXELS)). If this is set to "false", set PIL.Image.MAX_IMAGE_PIXELS = None in backend default vision handler for large image payload. 
```

Comment:
Hi @lxning, could you add more details on how the `limit_max_image_pixels`, fixes the related issue #1120 ? Or, at least aids in the resolution of the issue? [Request to add details in the PR description]. Is the idea that a user should be able to limit the size of the input image?

Reply:
the detailed information is provided in the above link